sql/parser/create.go

```diff
@@ -75,6 +76,17 @@ func (p *Parser) parseIfNotExists() (bool, error) {
 	return true, nil
 }
 
+func (p *Parser) parseField(fc *database.FieldConstraint) (err error) {
```

Comment:
How about `parseFieldDefinition` or something like that? I feel like `parseField` is too broad and doesn't really describe what the function is doing.